### PR TITLE
Privacy notice copy changes

### PR DESCRIPTION
**Pull request by tmckernan**
Screen Shot 2019-08-19 at 12 31 27

### CODE REVIEW

**Comment by tmckernan** [comment]
Screen Shot 2019-08-19 at 12 54 19

File: app/views/about/privacy.en.html.erb [hunk 1]
```diff
@@ -5,8 +5,34 @@
...
         <h1 class="govuk-heading-xl">Privacy notice</h1>
-        <p class="govuk-body">This privacy notice explains what the Ministry of Justice (MOJ) means when we talk about personal data, why we ask for this information about you and what we do with it when you use the ‘<%= service_name -%>’ online service.</p>
-        <p class="govuk-body">TODO: Add in appropriate content</p>
+        <p class="govuk-body">This service is completely anonymous to use. Users are not asked for any personal information that could be used to identify them (such as their name, address or prisoner number).</p>
+
+        <h2 class="govuk-heading-m">Telling us about your experience of using this service</h2>
+        <p class="govuk-body">We handle data differently if you choose to give feedback on your experience of using the service.</p>
+        <p class="govuk-body">If you have chosen to be contacted about your feedback, <a class="govuk-link" rel="external" target="_blank" href="https://apply-to-court-about-child-arrangements.service.justice.gov.uk/about/privacy_consent">find out how long we keep your data, your rights and how to contact us.</a></p>
```
Comment (zheileman) [inline app/views/about/privacy.en.html.erb]: This is C100 !!

**Comment by tmckernan** [comment]
Screen Shot 2019-08-19 at 14 26 34

File: app/views/about/privacy.en.html.erb [hunk 2]
```diff
@@ -5,8 +5,30 @@
...
+          <address class="adr">
+            <div class="extended-address">Information Commissioner's Office,</div>
+            <span class="extended-address">Wycliffe House,</span>
+            <span class="street-address">Water Lane,</span>
+            <div class="locality">Wilmslow,</div>
+            <span class="region">Cheshire,</span>
+            <span class="postal-code">SK9 5AF</span>
+          </address>
+
+          <div class="util_mt-medium">
```
Comment (zheileman) [inline app/views/about/privacy.en.html.erb]: Ok this is almost ready.
Minor details:

* The class `util_mt-medium` doesn't exist in the new design system. We can replace it for, more or less the equivalent, `govuk-!-margin-top-3`
* Let's add the `govuk-link` class to the ICO link so it is the right shade of blue.
* Optional: we might need to override the `address` default italic styling, as I seem to remember italic doesn't play well in accessibility tests. This probably will need to be confirmed with Matt and Simon, for now we can leave it as italic (browsers' default) but can you confirm with them please.

**Comment by tmckernan** [comment]
Screen Shot 2019-08-19 at 15 17 22

**Pull request merged**